docs/_docs/languages/hack.md

```diff
@@ -147,73 +141,82 @@ xdebug.remote_port = 9000
 
-Nuclide provides an HHVM toolbar. You can launch the toolbar from the
-[Nuclide toolbar](/docs/features/toolbar/#buttons) or from the
-[command-palette](/docs/editor/basics/#command-palette) with `Nuclide HHVM Toolbar: Toggle`.
+<!-- HHVM TOOLBAR WILL BE MOVING INTO TASK RUNNER -->
+Nuclide provides an HHVM toolbar. You can launch the toolbar from the [Nuclide toolbar](/docs/features/toolbar/#buttons) or from the
+[Command Palette](/docs/editor/basics/#command-palette) with `Nuclide HHVM Toolbar: Toggle`.
 
-> You have to have a Hack or PHP file open to successfully launch the toolbar.
+> You must have a Hack or PHP file open to successfully launch the toolbar.
 
 ![](/static/images/docs/feature-debugger-languages-hack-php-hhvm-toolbar.png)
 
-Now you can choose between debugging a webserver or a script.
+You can choose between debugging a webserver or a script.
 
 ![](/static/images/docs/feature-debugger-languages-hack-php-hhvm-toolbar-webserver-script.png)
 
 Set [breakpoints](/docs/features/debugger/#basics__breakpoints) in your code.
 
-Once you decide the type of debugging you plan to do, click `Attach`. This will bring up the
-debugger UI and stop at the first breakpoint you set.
+Once you decide the type of debugging you plan to do, click **Attach** (webserver) or **Launch** (script). This opens the
+Debugger Controls and stops at the first breakpoint.
 
 You can then follow the [basic debugging information](/docs/features/debugger/#basics) provided
 above and use the additional features of the Output Window, Evaluation and other HHVM-specific
 debugging settings as described below to debug your code.
 
-Here is an example of debugging a running HHVM server:
-
-![](/static/images/docs/feature-debugger-languages-hack-php-server-debugging.png)
+![](/static/images/docs/feature-debugger-languages-hack-debugging.png)
 
 In both the script and server launching/attaching scenarios, the line at which you've set a
 breakpoint will highlight in blue when the breakpoint is hit. When this happens, execution of your
-code is paused and you can use the debugger to step, evaluate expressions, inspect the current
-callstack, etc.
+code is paused and you can use the Debugger Controls to step, evaluate expressions, inspect the current
+call stack, etc.
 
 ### Debugging: Command-Line
 
-You can also debug without using the HHVM Toolbar by debugging directly from the command-line.
+You can also debug directly from the command-line.
 
-1. Have a PHP or Hack file active in the [editing area](/docs/editor/basics/#editing-area).
-2. `cmd-shift-Y` to bring up the process attachment UI.
-3. Choose the remote server that has your project and HHVM installed from the attachment UI.
-It should look similar to:
+1. Have a PHP or Hack file active in the [Editing Area](/docs/editor/basics/#editing-area).
+2. Click on the **Toggle Debugger** icon in the [Nuclide toolbar](/docs/features/toolbar/#buttons) or press `Cmd-Shift-Y` to bring up the Debugger Selection window.
```

Comment:
Can we change this from `cmd+shift+y` to `cmd+shift+a` (which is another keyboard shortcut that does the same thing).  I'd like to remove the cmd+shift+y binding